src/language-graphql/printer-graphql.js

```diff
@@ -24,10 +24,24 @@ function genericPrint(path, options, print) {
 
   switch (n.kind) {
     case "Document": {
-      return concat([
-        join(concat([hardline, hardline]), path.map(print, "definitions")),
-        hardline
-      ]);
+      const parts = [];
+      let i = 0;
+      path.map(pathChild => {
```

Comment:
The index is provided as the second parameter to `.map` so no need to manually count.